java-vtl-parser/src/main/antlr4/no/ssb/vtl/parser/VTL.g4

```diff
@@ -20,7 +20,9 @@
 grammar VTL;
 start : statement+ EOF;
 
-statement : assignment | expression ;
+statement : assignment ;
```

Comment:
No we would have if/else blocks at some point I think. I removed  the expression since it did not make any changes to the way test-suite.vtl is parsed.